timemachine/fe/dummy.py

```diff
@@ -269,28 +268,7 @@ def identify_dummy_groups(bond_idxs, core):
     #
     # in this case, we want the dummy atoms to be in different groups.
 
-    dummy_groups = []
-
-    for dg in cc:
-        anchor_membership = []
-        dg = list(dg)
-        for dummy in dg:
-            # identify membership of the dummy
-            anchors = identify_root_anchors(bond_idxs, core, dummy)
-            # get distance to anchor
-            dists = []
-            for a in anchors:
-                dists.append(nx.shortest_path_length(g, source=dummy, target=a))
-            anchor_membership.append(np.argmin(dists))
-
-        anchor_kv = defaultdict(set)
-        for idx, anchor in enumerate(anchor_membership):
-            anchor_kv[anchor].add(dg[idx])
-
-        for v in anchor_kv.values():
-            dummy_groups.append(v)
-
-    return dummy_groups
+    return cc
```

Comment:
The code below was deleted because it's no longer relevant, and was just plain buggy.

Reply:
remove comment above too

Reply:
willfix